test/test_main.py

```diff
@@ -46,10 +50,31 @@ def test_command_lib_install():
     result_2 = run_command('lib install {}'.format(' '.join(libs)))
     assert result_2.ok
 
+def test_command_lib_update_index():
+    result = run_command('lib update-index')
+    assert 'Updating index: library_index.json downloaded' == result.stdout.splitlines()[-1].strip()
 
 def test_command_lib_remove():
     libs = ['\"AzureIoTProtocol_MQTT\"', '\"CMMC MQTT Connector\"', '\"WiFiNINA\"']
     result = run_command('lib uninstall {}'.format(' '.join(libs)))
+    assert result.ok
+
+@pytest.mark.slow
+def test_command_lib_search():
+    result = run_command('lib search')
+    out_lines = result.stdout.splitlines()
+    libs = []
+    # Create an array with just the name of the vars
+    for line in out_lines:
+        if 'Name: ' in line:
+            libs.append(line.split()[1].strip('\"'))
+    number_of_libs = len(libs)
+    # It would be strange to have less than 2000 Arduino Libs published
+    assert number_of_libs > 2000
```

Comment:
any way to avoid using a magic number? Can we search for a specific query instead and check for the expected number of items?

Reply:
Libraries are users' maintained and subject to changes or removal from the index. I would not put that in the tests. Maybe @cmaglie can help here. There could be some never changing libs

Reply:
solved, I added a check based on some libs found as output that are not going to change names in the next few years.